src/algebra/group_power/order.lean

```diff
@@ -341,57 +341,46 @@ by simpa only [sq] using abs_mul_abs_self x
 theorem abs_sq (x : R) : |x ^ 2| = x ^ 2 :=
 by simpa only [sq] using abs_mul_self x
 
-theorem sq_lt_sq (h : |x| < |y|) : x ^ 2 < y ^ 2 :=
-by simpa only [sq_abs] using pow_lt_pow_of_lt_left h (abs_nonneg x) (1:ℕ).succ_pos
+theorem sq_lt_sq : x ^ 2 < y ^ 2 ↔ |x| < |y| :=
```

Comment:
Shouldn't this be called `sq_lt_sq_iff` or something like that?

Reply:
We're not very consistent about `iff`s. E.g., we have `sub_pos`. Formally, this should be something like `sq_lt_sq_iff_abs_lt_abs` but this is to long.